doc/roadmap_2023.md

```diff
@@ -0,0 +1,193 @@
+# Roadmap for 2023
+
+2022 was an interesting year for Vello. We made good progress on a number of fronts, solving some really tricky problems, and also did a rewrite into WGSL, but did not ship a usable renderer. Oh, and we renamed the project from piet-gpu as well.
+
+2023 promises to be even more interesting. With some focus we can ship a usable renderer. The core team (Raph Levien and Chad Brokaw, with help from Arman Uguray) will be able to do most of the tasks listed here. Help from the community could speed things up or add even more features. We also rely heavily on the wgpu community to build solid GPU infrastructure for our uses, and also expect a nontrivial amount of integration work as people adopt Vello for their projects.
+
+Large parts of the [piet-gpu vision] are still valid, but of course there are updates in the two years since it was written.
+
+This roadmap is in roughly chronological in order of when we expect to complete the work.
```

Comment:
Optional: I think this could benefit from an executive summary. Give me the bullet point features and why they matter, tell me what sort of projects vello might make sense for by eoy. Right now there's a lot of text to get through to get the higher level sense of where we're going.